assets/csidriveroperators/shared-resource/05_clusterrole.yaml

```diff
@@ -88,8 +88,15 @@ rules:
   resources:
   - customresourcedefinitions
   verbs:
-  - list
```

Comment:
`list`,`get`, and `watch` are IMO safe to grant for all CRDs.

Granting list to all CRDs is needed to avoid issues with listers - a serviceaccount with restrictions on list needs to provide the name in a field selector, which is cumbersome.

See https://kubernetes.io/docs/reference/access-authn-authz/rbac/#referring-to-resources

Reply:
We are using the fieldSelector (https://github.com/openshift/csi-driver-shared-resource-operator/blob/master/pkg/operator/starter.go#L88-L90), and honestly I think we are being slack in our permissions granting (probably throughout openshift honestly) to save a line of code.

Reply:
> We are using the fieldSelector (https://github.com/openshift/csi-driver-shared-resource-operator/blob/master/pkg/operator/starter.go#L88-L90), and honestly I think we are being slack in our permissions granting (probably throughout openshift honestly) to save a line of code.

ack, then :+1: